docs/source/intel/configuration.mdx

```diff
@@ -0,0 +1,17 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Trainer
+
+## IncTrainer
+
+[[autodoc]] intel.neural_compressor.trainer_inc.IncTrainer
```

Comment:
There seems to be a mismatch with `docs/source/intel/trainer.mdx`

Reply:
Yes, I wasn't sure whether to use the same hierarchy as we have in the `onnxruntime` docs. Happy to nest it though so it lines up with the source code!

Reply:
What I meant is that `IncTrainer` is displayed in `configuration.mdx` while`IncOptimizer` is displayed in `trainer.mdx`

Reply:
What I meant is that `IncTrainer` is displayed in `configuration.mdx` while`IncOptimizer` is displayed in `trainer.mdx`

Reply:
Good catch, fixed now!